src/Adapter/MSTest.TestAdapter/Execution/TestExecutionManager.cs

```diff
@@ -451,7 +462,10 @@ private void ExecuteTestsWithTestRunner(
             // Run single test passing test context properties to it.
             IDictionary<TestProperty, object?> tcmProperties = TcmTestPropertiesProvider.GetTcmProperties(currentTest);
             Dictionary<string, object?> testContextProperties = GetTestContextProperties(tcmProperties, sourceLevelParameters);
-            UnitTestResult[] unitTestResult = testRunner.RunSingleTest(unitTestElement.TestMethod, testContextProperties);
+
+            // testRunner is in a different app domain. We cannot pass the testExecutionRecorder directly.
```

Comment:
I would change the wording to `could be in a different AppDomain`.

Reply:
```suggestion
            // testRunner could be in a different AppDomain. We cannot pass the testExecutionRecorder directly.
```